pynxtools/nyaml2nxdl/nyaml2nxdl.py

```diff
@@ -148,33 +148,44 @@ def split_name_and_extension(file_name):
 
 @click.command()
 @click.option(
+    '-i',
     '--input-file',
     required=True,
     prompt=True,
     help='The path to the XML or YAML input data file to read and create \
 a YAML or XML file from, respectively.'
 )
 @click.option(
+    '-a',
     '--append',
     help='Parse xml file and append to base class, given that the xml file has same name \
 of an existing base class'
 )
 @click.option(
+    '-c',
     '--check-consistency',
     is_flag=True,
     default=False,
-    help=('Check wether yaml or nxdl has followed general rules of scema or not'
+    help=('Check wether yaml or nxdl has followed general rules of schema or not'
           'check whether your comment in the right place or not. The option render an '
           'output file of the same extension(*_consistency.yaml or *_consistency.nxdl.xml)')
 )
 @click.option(
+    '-v',
     '--verbose',
     is_flag=True,
     default=False,
     help='Print in standard output keywords and value types to help \
 possible issues in yaml files'
 )
-def launch_tool(input_file, verbose, append, check_consistency):
+@click.option(
+    '-s',
+    '--self-closing-tag-space',
+    is_flag=True,
+    default=False,
+    help='Ensure there is a single space character before all self-closing XML tag'
+)
```

Comment:
Do we need this as a flag? Wouldn't it be better to have this as the default?

Reply:
I will change and flip the logic.